guacamole/src/main/webapp/app/login/loginModule.js

```diff
@@ -23,4 +23,4 @@
 /**
  * The module for the login functionality.
  */
-angular.module('login', ['element']);
+angular.module('login', ['element', 'navigation']);
```

Comment:
Very true. I agree it's nearly 100% likely we will be specifying (and using) this before the story is done.

It feels a bit weird to have to remember to remove this if we don't actually end up using it, though. It may have to be part of the overall set of changes, but it doesn't seem like it should be in this change... right?

It would probably be silly to add another commit just to remove this, but perhaps we could amend this commit so our history makes sense, and we don't have to worry about it during the rest of development?